app/assets/stylesheets/administrate/components/_buttons.scss

```diff
@@ -34,3 +34,8 @@
     }
   }
 }
+
+.scope-buttons {
```

Comment:
Gotcha! No worries.

If you you’d like, I’m more than happy to take on the styles side.